src/vm/tieredcompilation.cpp

```diff
@@ -413,6 +425,10 @@ void TieredCompilationManager::ActivateCodeVersion(NativeCodeVersion nativeCodeV
         {
             CodeVersionManager::TableLockHolder lock(pCodeVersionManager);
             hr = ilParent.SetActiveNativeCodeVersion(nativeCodeVersion, TRUE);
+            LOG((LF_TIEREDCOMPILATION, LL_INFO10000, "TieredCompilationManager::ActivateCodeVersion Method=0x%x (%s::%s), code version id=0x%x. [Suspended] SetActiveNativeCodeVersion ret=0x%x\n",
```

Comment:
The format character for the pMethodDesc is incorrect (%x, should be %p and probably formatted as %pM like in the preexisting log call below)

Reply:
Ah thanks, I had spotted that elsewhere but forgot to come back. I suspect you'll find the same thing in some of the other recent PRs that I'll need to fix up.